mempool/reactor.go

```diff
@@ -233,9 +226,12 @@ func (memR *Reactor) broadcastTxRoutine(peer p2p.Peer) {
 
 		// ensure peer hasn't already sent us this tx
 		if _, ok := memTx.senders.Load(peerID); !ok {
-			// send memTx
-			msg := &TxMessage{Tx: memTx.tx}
-			success := peer.Send(MempoolChannel, cdc.MustMarshalBinaryBare(msg))
+			msg := gogotypes.BytesValue{Value: []byte(memTx.tx)}
+			bz, err := msg.Marshal()
+			if err != nil {
+				// TODO: question bubble up the error or ignore it?
```

Comment:
The previous behaviour was to panic in case of an error `MustMarshalBinaryBare`.